src/components/option-lab/quantity-input.tsx

```
@@ -9,43 +9,71 @@ export const QuantityInput = () => {
+    setValue('quantity', formattedValue);
   };
 
   return (
     <div className="space-y-2">
       <div className="flex items-center gap-2">
-        <label className="text-sm font-medium text-white/80">Quantity</label>
-        <Tooltip content="The number of option contracts to sell. Fractional quantities are supported.">
-          <Info className="w-4 h-4 text-white/40 cursor-help" />
+        <label className="text-sm font-medium text-white/60">
+          Contract Quantity <span className="opacity-50">(1 = 100 Underlying Tokens)</span>
+        </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)